.github/workflows/linuxtests.yml

```diff
@@ -0,0 +1,58 @@
+name: Linux Tests
+
+on:
+  push:
+    branches:
+      - master
+      - maintenance/**
+  pull_request:
+    branches:
+      - master
+      - maintenance/**
+
+jobs:
+  Python_3.7_Debug
+    name: Python 3.7 Debug
+    runs-on: ubuntu-18.04
+    env:
```

Comment:
I wouldn't bother with environment variables to start with, just use them in the lines when they're required. e.g. Line 25: `virtualenv --python=python3.7-dbg venv`

Reply:
I'll clean up once we actually make it working the rest I can do [ci skip] anyways.